backend/apps/schemes/services/scheme_item_service.py

```
@@ -1,27 +1,47 @@
+        RequiredFieldsRule(["scheme", "content_type", "object_id"], "SchemeItem"),
+    ]
 
     # ---------------------------------------------------------------------
     # Basic CRUD Operations
     # ---------------------------------------------------------------------
 
-    @staticmethod
+    @classmethod
     @transaction.atomic
-    def scheme_item_create(*, scheme_item_data: dict, user=None):
+    def scheme_item_create(cls, *, scheme_item_data: dict, user=None):
```

Ruff (0.14.3)

Unused class method argument: `user`

(ARG003)